tools/run_tests/python_utils/start_port_server.py

```diff
@@ -92,16 +93,16 @@ def start_port_server(port_server_port):
     waits = 0
     while True:
       if waits > 10:
-        print('killing port server due to excessive start up waits')
+        logging.warning('killing port server due to excessive start up waits')
         port_server.kill()
       if port_server.poll() is not None:
-        print('port_server failed to start')
+        logging.error('port_server failed to start')
         # try one final time: maybe another build managed to start one
         time.sleep(1)
         try:
           urllib.request.urlopen('http://localhost:%d/get' % port_server_port,
                           timeout=1).read()
-          print('last ditch attempt to contact port server succeeded')
+          logging.info('last ditch attempt to contact port server succeeded')
           break
         except:
           traceback.print_exc()
```

Comment:
(ah, yes, gotcha!) While we are here... does the logic seem wrong to you too? The `try` will break out of the `while True` if `urlopen` doesn't raise, but then the message in the `except` talks about a success and reads the result off a file. At a minimum, the logging message shouldn't much talk about success? @ctiller , as the author, is this just a mistake or am I missing something here?